private/pkg/git/git.go

```diff
@@ -135,3 +139,22 @@ type ListFilesAndUnstagedFilesOptions struct {
 	// is being applied to.
 	IgnorePathRegexps []*regexp.Regexp
 }
+
+// GetDefaultBranch returns the repository's default branch name. It attempts to read it from the
+// `.git/refs/remotes/origin/HEAD` file, and expects it to be pointing to a branch also in the
+// `origin` remote.
+func GetDefaultBranch(gitDirPath string) (string, error) {
```

Comment:
This appears unused?

Reply:
Oh! I pulled this in and no longer needed it, and didn't realise i didn't clear this file before committing. Yes, that is correct, will be deleting.